server/controller/src/main/java/ai/starwhale/mlops/schedule/k8s/K8sTaskScheduler.java

```diff
@@ -148,11 +149,6 @@ private void deployTaskToK8s(Task task) {
     private Map<String, ContainerOverwriteSpec> buildContainerSpecMap(Task task) {
         // TODO: use task's resource needs
         Map<String, ContainerOverwriteSpec> ret = new HashMap<>();
-        k8sJobTemplate.getInitContainerTemplates().forEach(templateContainer -> {
```

Comment:
just delete init containers in the template works too.

Reply:
done